flixel/addons/display/FlxRuntimeShader.hx

```diff
@@ -1,6 +1,6 @@
 package flixel.addons.display;
 
-#if (FLX_DRAW_QUADS && !flash)
+#if (!nme && !flash)
```

Comment:
this change makes sense but it was setup wrong before and we should fix it while we're here.

```suggestion
#if nme
    #error "FlxRuntimeShader is not available with nme"
#elseif flash
    #error "FlxRuntimeShader is not available when targeting flash"
#else
```

Reply:
also note that the `#else` can just be an `#end` to simplify (it doesn't seem to compile the rest of the file once `#error` is reached

Reply:
I've been using these for all my native C++ libs, it's pretty useful but i never though about using it here.